v2/internal/reconcilers/azure_generic_arm_reconciler.go

```diff
@@ -553,8 +539,7 @@ func (r *AzureDeploymentReconciler) handlePollerFailed(ctx context.Context, err
 	}
 
 	// We probably just modified spec so don't need to requeue this
-	needRequeue := !hasResumeToken
-	return ctrl.Result{Requeue: needRequeue}, nil
+	return ctrl.Result{Requeue: true}, nil
```

Comment:
Comment says "probably don't need to", but then we always do. Is that just in case?  Perhaps clarify the comment?

Reply:
I should remove this comment - it no longer applies